source/backtraceClient.ts

```diff
@@ -22,6 +23,7 @@ export class BacktraceClient {
   public readonly attributes: { [index: string]: any };
 
   private _backtraceApi: BacktraceApi;
+  private _backtraceMetrics: BacktraceMetrics;
```

Comment:
hmm If we don't expose backtraceMetrics instance here in the client, how other developers can add unique/summed events?

Reply:
Making public, my confusion I thought we were not letting developers call metrics manually.